### PR TITLE
pgx upgraded to v4.10.1

### PR DESCRIPTION
**Pull request by porfirion**

### CODE REVIEW

**Comment by porfirion** [comment]
@omnilight @dedefer @vasileknik76 @grazor review pls

**Comment by omnilight** [comment]
Ок :)

File: postgres.go [hunk 1]
```diff
@@ -106,11 +106,12 @@ func (s *PostgresMutex) lock() error {
 			time.Sleep(jitter)
 		}
 
-		if s.tx, err = s.client.Begin(); err != nil {
+		s.tx, err = s.client.Begin(context.TODO())
```
Comment (omnilight) [inline postgres.go]: if we are not planning to add contexts later, just use `context.Background()`

**Review by omnilight — APPROVED** [review]

File: postgres.go (hunk 1 above)
Comment (porfirion) [reply postgres.go]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# v0.0.5
```
Comment (lukashes) [inline CHANGELOG.md]:
```suggestion
# Unreleased
```

This version already occupied. 

There is a github releases used for tracking changes.

**Review by lukashes — CHANGES_REQUESTED** [review]
Thank you for PR. It looks, great. I added one minor comment regarding tracking changes.

**Pull request merged**